contracts/src/v0.8/functions/dev/v1_X/FunctionsRouter.sol

```diff
@@ -19,7 +19,7 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
   using FunctionsResponse for FunctionsResponse.FulfillResult;
 
   // solhint-disable-next-line chainlink-solidity/all-caps-constant-storage-variables
-  string public constant override typeAndVersion = "Functions Router v1.0.0";
+  string public constant override typeAndVersion = "Functions Router v2.0.0";
```

Comment:
Which version do we want to use @justinkaseman ?

Reply:
Since these small two changes do actually change change the compiled bytecode of the contract we incrementing to V2 is correct here.

If some some reason we decide not to use V2 Router on the next deploy we can reference the `contracts/src/v0.8/functions/v1_0_0` code